consensus/src/dag/dag_store.rs

```diff
@@ -46,13 +42,17 @@ pub struct Dag {
     author_to_index: HashMap<Author, usize>,
     storage: Arc<dyn DAGStorage>,
     initial_round: Round,
+    epoch_state: Arc<EpochState>,
+
+    highest_committed_anchor_round: Round,
 }
 
 impl Dag {
     pub fn new(
         epoch_state: Arc<EpochState>,
         storage: Arc<dyn DAGStorage>,
         initial_round: Round,
+        highest_committed_anchor_round: Round,
```

Comment:
do we need both initial round and highest committed round? seem like the same thing?

Reply:
we need the initial round for the window. We could pass the window config, but this seems better for testing.

Reply:
we can derive that from the highest committed round? i.e. highest committed round - WINDOW

Reply:
yes, that is correct.  we will get rid off WINDOW and then we need to pass in the config anyhow right? so, its either we pass the window as a param or the initial round as a param. are you thinking its inconsistent?

Reply:
oh i see, I think pass in the window size is cleaner, maintaining two round is a little confusing what's the relationship between them